controllers/app.controller.js

```diff
@@ -1,19 +1,27 @@
-const {getAllTopics} = require("../models/app.model")
+const {getAllTopics, getArticleById} = require("../models/app.model")
 const allEndpoints = require("../endpoints.json")
 
 
 function topicsController(request, response, next) {
     return getAllTopics().then((data) => {
         return response.status(200).send({topics : data});
     })
-    .catch(err => {
-        next(err);
-    })
 }
 
 function apiController(request, response, next) {
     response.status(200).send(allEndpoints);
 }
 
-module.exports = {topicsController, apiController}
+function articleController(request, response, next) {
```

Comment:
Variable naming could be better in this file - what does this function actually do?